empty/routes/users.js

```diff
@@ -18,11 +18,16 @@ router.post('/login', function ( req, res, next ) {
 
   db.retrieveUser( { user_name: user_name } )
     .then( user => {
-      if (user[0].password === password) {
-        res.redirect( '/game' )
+      if ( user[0] ) {
+        if ( user[0].password === password ) {
+          res.redirect('/game')
+        }
+        else {
+          res.send('wrong password you, but')
```

Comment:
We do not want to encourage hackers to keep guessing - a simple failure (perhaps with a `401` response) is considered best practice.